### PR TITLE
[PyOV] Add `torchvision` to OpenVINO preprocessing converter docs

### PR DESCRIPTION
**Pull request by p-wysocki**
### Details:
 - Add documentation for `torchvision` to OpenVINO preprocessing converter

### Tickets:
 - 123966

### CODE REVIEW

File: src/bindings/python/src/openvino/preprocess/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- `transforms.ToTensor`
+- `transforms.CenterCrop`
+- `transforms.Resize`
+
+## Example usage
+
+```python
+preprocess_pipeline = torchvision.transforms.Compose(
+    [
+        torchvision.transforms.Resize(256, interpolation=transforms.InterpolationMode.NEAREST),
```
Comment (AlexKoff88) [inline src/bindings/python/src/openvino/preprocess/README.md:25]: Do we support Bilinear interpolation from Pillow library? I noticed many times that it is a default one.

Comment (p-wysocki) [reply src/bindings/python/src/openvino/preprocess/README.md:25]: I did some digging and it turned out that while we have Pillow interpolation modes implemented in C++, we do not have Python bindings for them. Let's continue this topic in ticket 124943 and its Good First Issue https://github.com/openvinotoolkit/openvino/issues/21042.

**Comment by mlukasze** [comment]
is there anything else reviewers expects to be changed or can we merge it?

**Review by jiwaszki — APPROVED** [review]

File: src/bindings/python/src/openvino/preprocess/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Torchvision to OpenVINO preprocessing converter
+
+Torchvision to OpenVINO preprocessing converter allows you to use an existing `torchvision.transforms` object to automatically translate it to OpenVINO preprocessing. It is then being embedded into the model, resulting in better inference performance.
```
Comment (kblaszczak-intel) [inline src/bindings/python/src/openvino/preprocess/README.md:3]:
```suggestion
The Torchvision to OpenVINO preprocessing converter enables the use of an existing `torchvision.transforms` object to automatically translate it to OpenVINO preprocessing. It is then being embedded into the model, resulting in better inference performance.
```

File: src/bindings/python/src/openvino/preprocess/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+
+## Supported transforms
+
+Current torchvision to OpenVINO preprocessing converter does not support all torchvision transforms.
```
Comment (kblaszczak-intel) [inline src/bindings/python/src/openvino/preprocess/README.md:8]:
```suggestion
Currently, the torchvision to OpenVINO preprocessing converter does not support all torchvision transforms.
```

**Review by kblaszczak-intel — APPROVED** [review]

**Review by akuporos — APPROVED** [review]

**Pull request merged**